opba-protocols/sandboxes/xs2a-sandbox/src/test/java/de/adorsys/opba/protocol/xs2a/testsandbox/BasicTest.java

```diff
@@ -52,17 +54,37 @@ void testEnvStartsUp() {
             .statusCode(500);
     }
 
+    /**
+     * Not really a test, but just launches entire old version sandbox for you.
+     * If run with intellij 2018.3 please set VM option to -DSTART_SANDBOX_OLD=true and environment-variable ENABLE_HEAVY_TESTS=true
+     */
+    @Test
+    @SneakyThrows
+    @EnabledIfSystemProperty(named = "START_SANDBOX_OLD", matches = TRUE_BOOL)
+    void startTheSandbox() {
+        startSandboxWithJars();
+
+        assertThat(SandboxApp.values()).extracting(it -> ctx.getLoader().get(it)).isNotNull();
+
+        // Loop forever.
+        await().forever().until(() -> false);
+    }
+
     /**
      * Not really a test, but just launches entire sandbox for you.
      * If run with intellij 2018.3 please set VM option to -DSTART_SANDBOX=true and environment-variable ENABLE_HEAVY_TESTS=true
      */
     @Test
     @SneakyThrows
     @EnabledIfSystemProperty(named = "START_SANDBOX", matches = TRUE_BOOL)
-    void startTheSandbox() {
-        executor.awaitForAllStarted();
+    void startTheSandboxInDocker() {
+        File directory = new File(getClass().getClassLoader().getResource(SANDBOX_DOCKER_RUNNER).toURI()).getParentFile();
 
-        assertThat(SandboxApp.values()).extracting(it -> ctx.getLoader().get(it)).isNotNull();
+        ProcessBuilder builder = new ProcessBuilder().inheritIO();
```

Comment:
Why not using TestContainers:
https://www.testcontainers.org/modules/docker_compose/

it has benefit of having ryuk daemon which kills everything on failure or on test stop

Reply:
Done